src/app/features/search/multiple-search-results/multiple-search-results.component.spec.ts

```diff
@@ -56,8 +58,25 @@ describe('MultipleSearchResultsComponent', () => {
       expect(component).toBeTruthy();
     });
 
-    it('should navigate back', fakeAsync(() => {
+    it('should return not found error and create link when searchResults is null', async () => {
+      const button = fixture.debugElement.query(By.css('.govuk-link'));
+      expect(button).toBeTruthy();
+    });
+
+    it('should navigate back when searchResults not null', fakeAsync(() => {
       const navigateBackSpy = jest.spyOn(component, 'navigateBack');
```

Comment:
Do you need this navigateBackSpy? it doesn't appear to be being used

Reply:
I think we need it. We expect navigateBackSpy function could be called whenever back link is clicked.